app/assets/javascripts/discourse-plugins/index.js

```diff
@@ -45,18 +45,18 @@ function unColocateConnectors(tree) {
         match.groups.extension === "hbs" &&
         match.groups.prefix.split("/").pop() !== "templates"
       ) {
-        const { prefix, outlet, name } = match.groups;
-        return `${prefix}/templates/connectors/${outlet}/${name}.hbs`;
+        const { prefix, connector } = match.groups;
+        return `${prefix}/templates/connectors/${connector}.hbs`;
       }
       if (
         match &&
         match.groups.extension === "js" &&
         match.groups.prefix.split("/").pop() === "templates"
```

Comment:
Maybe this should be `match.groups.prefix.endsWith("/templates")` since that is what we expect later on line 58.

Reply:
```suggestion
        match.groups.prefix.endsWith("/templates")
```